recipes/tbb/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+export MACOSX_DEPLOYMENT_TARGET=10.10
```

Comment:
Is 10.10 the lowest you could go?  I understand 10.7 doesn't work, but what about 10.8?

Backwards compatibility is less of a concern, but still something of a concern on OS X.